src/ert/gui/plottery/plots/ccsp.py

```diff
@@ -41,7 +41,7 @@ def plotCrossCaseStatistics(figure, plot_context, case_to_data_map, _observation
         "p67": {},
         "p90": {},
     }
-    for case_index, (case, data) in enumerate(case_to_data_map.items()):
+    for case_index, data in enumerate(case_to_data_map.values()):
```

Comment:
why do we need `data` here? Is the usage of `_` correct in this case?

```suggestion
    for case_index, _ in enumerate(case_to_data_map.values()):
```

Reply:
`_ ` is typical to use for variables that you don't really need. Since `enumerate` yields tuples, we have this dummy variable name to get the tuple unpacked.